gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/CopySource.java

```diff
@@ -214,6 +214,7 @@ public List<WorkUnit> getWorkunits(final SourceState state) {
       failJobIfAllRequestsRejected(allocator, prioritizedFileSets);
 
       String filesetWuGeneratorAlias = state.getProp(ConfigurationKeys.COPY_SOURCE_FILESET_WU_GENERATOR_CLASS, FileSetWorkUnitGenerator.class.getName());
+      boolean isWUFastFailOverEnabled = state.getPropAsBoolean(ConfigurationKeys.WORK_UNIT_FAST_FAIL_ENABLED, true);
```

Comment:
Nit: Maybe store the default as a static variable? e.g. `DEFAULT_WORK_UNIT_FAST_FAIL_ENABLED`

Reply:
And do we want the default to be true? The description mentions possibility of breaking user flows, so we should be careful about if this feature is opt-in or opt-out.

Reply:
Also, the term "failover" implies there is a graceful backup plan right? Maybe you meant just fast fail.

Reply:
I also agree that the default of 'true' is justified as well. I'd rather we not have configuration drift and for a vast majority of use cases having this enabled is probably the correct behavior, though having a setting to turn this off is useful in case we missed anything.

Reply:
I agree with all your comments and feedback. I have created a static default variable and also renamed the boolean variable. I have updated my PR with the changes.